server/src/test/java/org/elasticsearch/search/profile/query/ProfileCollectorManagerTests.java

```diff
@@ -142,110 +137,4 @@ public void testManagerWithSearcher() throws IOException {
             assertTrue(result.getTime() > 0);
         }
     }
-
-    public void testManagerWithChildren() throws IOException {
```

Comment:
These were testing support for children collectors, which ProfileCollectorManager no longer support